storybookjs/storybook
# Remove `textTransform: 'capitalize'` from component `bar/button`

Issue:
Using addon-contexts, I tried adding a context button to the top bar with the title "iOS/Android". I believe due to an unnecessarily opinionated style rule applied to the bar/button component, this is transformed via CSS to "IOS/Android" (capital i on iOS). Removing the `textTransform: 'capitalize'` rule from this component should (I think?) eliminate this problem.

## What I did
Remove `textTransform: 'capitalize'` from `lib/components/src/bar/button.ts`.

## How to test

- Is this testable with Jest or Chromatic screenshots?
Sorry, not sure.

- Does this need a new example in the kitchen sink apps?
I don't think so....?

- Does this need an update to the documentation?
No.

If your answer is yes to any of these, please make sure to include it in your PR.
